examples/src/content/jcr_root/content/core-components-examples/library/download/.content.xml

```diff
@@ -0,0 +1,256 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        cq:lastModified="{Date}2019-05-27T23:24:13.683+02:00"
+        cq:lastModifiedBy="admin"
+        cq:tags="[core-components-examples:component-type/basic]"
+        cq:template="/conf/core-components-examples/settings/wcm/templates/content-page"
+        jcr:description="Provide a download item"
+        jcr:primaryType="cq:PageContent"
+        jcr:title="Download"
+        sling:resourceType="core-components-examples/components/page">
+        <root
+            jcr:primaryType="nt:unstructured"
+            sling:resourceType="wcm/foundation/components/responsivegrid">
+            <responsivegrid
+                cq:styleIds="[1545309110033]"
+                jcr:created="{Date}2018-12-20T12:31:03.201Z"
+                jcr:createdBy="admin"
+                jcr:lastModified="{Date}2018-12-20T14:47:57.433Z"
+                jcr:lastModifiedBy="admin"
+                jcr:primaryType="nt:unstructured"
+                sling:resourceType="wcm/foundation/components/responsivegrid">
+                <text
+                    cq:styleIds="[1644862132301]"
+                    jcr:created="{Date}2019-05-21T12:23:37.787+02:00"
+                    jcr:createdBy="admin"
+                    jcr:lastModified="{Date}2019-05-21T12:23:47.906+02:00"
+                    jcr:lastModifiedBy="admin"
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="core/wcm/components/text/v2/text"
+                    text="&lt;h1>Download&lt;sub>v1&lt;/sub>&lt;/h1>"
+                    textIsRich="true"/>
+                <text_442449340
+                    jcr:created="{Date}2019-05-21T12:26:28.282+02:00"
```

Comment:
@pauli - this intro text is missing a style ID. 

`cq:styleIds="[1544762734201]"`